it/src/main/scala/com/wavesplatform/it/api/SyncHttpApi.scala

```diff
@@ -20,6 +22,13 @@ object SyncHttpApi extends Assertions{
     case _ => Assertions.fail(s"Expecting bad request")
   }
 
+  def assertBadRequestAndMessage2[R](f: => R, errorMessage: String): Assertion = Try(f) match {
```

Comment:
that doesn't look like proper function name :) please change it.
@DrBlast the existing one `assertBadRequest2` is not great either

Reply:
@ismagin if function name will be assertBadRequestAndMessageS or assertBadRequestAndMessageSync, it is ok?